### PR TITLE
Update getting_started_introduction.md

### PR DESCRIPTION
**Pull request by e11i0t23**
## Description

These are some changes to make the introduction clearer by including the addition of talking about vendor and manufacture sub folders. as well as add information of a couple other files found each keyboard. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/getting_started_introduction.md [hunk 1]
```diff
@@ -12,11 +12,17 @@ Within the folder `users` is a directory for each user. This is a place for user
 
 ### Keyboard Project Structure
 
-Within the folder `keyboards` and its subfolder `handwired` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard`. Within it you'll find the following structure:
+Within the folder `keyboards`, its subfolder `handwired` and its vendor and manufacture subdirectories e.g. `clueboard` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard/2x1800`. Within it you'll find the following structure:
 
 * `keymaps/`: Different keymaps that can be built
 * `rules.mk`: The file that sets the default "make" options. Do not edit this file directly, instead use a keymap specific `rules.mk`.
 * `config.h`: The file that sets the default compile time options. Do not edit this file directly, instead use a keymap specific `config.h`.
+* `info.json`: The file used for setting layout for qmk configurator see [Configurator Support](reference_configurator_support.md) for more information.
```
Comment (noroadsleft) [inline docs/getting_started_introduction.md]:
```suggestion
* `info.json`: The file used for setting layout for QMK Configurator. See [Configurator Support](reference_configurator_support.md) for more information.
```

File: docs/getting_started_introduction.md [hunk 2]
```diff
@@ -12,11 +12,17 @@ Within the folder `users` is a directory for each user. This is a place for user
...
+* `readme.md`: A brief overview of the keyboard.
+* `<keyboardName>.h`: This file is where the keyboard layout is defined against the keyboards physical matrix.
```
Comment (noroadsleft) [inline docs/getting_started_introduction.md]:
```suggestion
* `<keyboardName>.h`: This file is where the keyboard layout is defined against the keyboard's switch matrix.
```

File: docs/getting_started_introduction.md [hunk 3]
```diff
@@ -12,11 +12,17 @@ Within the folder `users` is a directory for each user. This is a place for user
...
+* `<keyboardName>.c`: This file is where you can find custom code for the keyboard.  
+
+For more information on project structure see [QMK Keyboard Guidelines](hardware_keyboard_guidelines.md).
```
Comment (noroadsleft) [inline docs/getting_started_introduction.md]:
```suggestion
For more information on project structure, see [QMK Keyboard Guidelines](hardware_keyboard_guidelines.md).
```

Comment (e11i0t23) [reply docs/getting_started_introduction.md]: Pesky little comma's. Glad someone can grammer

File: docs/getting_started_introduction.md [hunk 4]
```diff
@@ -12,11 +12,17 @@ Within the folder `users` is a directory for each user. This is a place for user
 
 ### Keyboard Project Structure
 
-Within the folder `keyboards` and its subfolder `handwired` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard`. Within it you'll find the following structure:
+Within the folder `keyboards`, its subfolder `handwired` and its vendor and manufacture subdirectories e.g. `clueboard` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard/2x1800`. Within it you'll find the following structure:
```
Comment (noroadsleft) [inline docs/getting_started_introduction.md]: Was gonna let this one go, but nope, it still bothers me (not your fault; it's been like this for a while).

```suggestion
Within the folder `keyboards`, its subfolder `handwired` and its vendor and manufacture subdirectories e.g. `clueboard` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard/2x1800`. Within it, you'll find the following structure:
```

**Pull request merged**